packages/debug-utils/index.js

```diff
@@ -532,16 +535,19 @@ var DebugUtils = {
   },
 
   formatCurrentInstruction: function (instruction) {
+    // return warning message if the debugger does not have the code for this contract
+    if (!instruction || !instruction.pc) {
+      return "WARNING: The debugger does not have the code for this contract.";
```

Comment:
Minor note: In other places that the debugger prints out warnings they get bolded, so you might want to apply some bold formatting here (`debug-utils` imports `chalk` so you can use that).